frontend/kesaseteli/employer/src/components/application/companyInfo/CompanyInfo.sc.ts

```diff
@@ -8,6 +8,10 @@ export const $CompanyInfoGrid = styled.div`
   padding-bottom: ${(props) => props.theme.spacing.s};
 `;
 
-export const $CompanyInfoHeader = styled.div``;
+export const $CompanyInfoHeader = styled.div`
+  font-weight: 500;
+`;
 
-export const $CompanyInfoCell = styled.div``;
+export const $CompanyInfoCell = styled.div`
+  font-size: 18px;
```

Comment:
I wonder if this could ne done with supported font sizes: https://github.com/City-of-Helsinki/yjdh/blob/develop/frontend/shared/src/styles/theme.ts
```
  fontSize: {
    body: {
      s: 'var(--fontsize-body-s)',
      m: 'var(--fontsize-body-m)',
      l: 'var(--fontsize-body-l)',
      xl: 'var(--fontsize-body-xl)',
    },
  },
```